### ISSUE TITLE
When trying to run the package operator on mac air m2 by the `make run` command getting error saying can't find `tls.crt` - no such file or directory.

### ISSUE THREAD

**Issue by bishalbera** [glasskube/glasskube#813]
**Describe the bug**
I was working on #764 . When running the package operator on mac air m2 its returning with an error saying `problem running manager	{"error": "open /var/folders/gs/6x7sqthj2wscyhddhzzdtwdw0000gn/T/k8s-webhook-server/serving-certs/tls.crt: no such file or directory"}`. However, when I did the same thing in my ubuntu it worked. After going through the logs I found out that when running the `make setup` command its saving the certificates in the `/tmp/k8s-webhook-server/serving-certs` . but when using the `make run` command its checking for the certs in ` /var/folders/gs/6x7sqthj2wscyhddhzzdtwdw0000gn/T/k8s-webhook-server/serving-certs/tls.crt` and there is no directory called `k8s-webhook-server` in the `T` folder.

**To Reproduce**
1. You should have a mac(m2)
2. Follow [this](https://github.com/glasskube/glasskube/blob/main/CONTRIBUTING.md#custom-package-repository) instruction .

**Expected behavior**
The operator should run successfully.

**Screenshots**
<img width="1710" alt="Screenshot 2024-06-15 at 9 55 25 AM" src="https://github.com/glasskube/glasskube/assets/123734227/5acc02aa-50c4-4c8d-a60c-3fee8b1d8fb2">
<img width="1710" alt="Screenshot 2024-06-15 at 9 57 55 AM" src="https://github.com/glasskube/glasskube/assets/123734227/61039ab4-37a6-4ef5-80c8-f7d6f3768795">
<img width="1710" alt="Screenshot 2024-06-15 at 9 59 39 AM" src="https://github.com/glasskube/glasskube/assets/123734227/13deff04-0858-4c1c-ad42-ca45ba09b8a2">

**Cluster Info (please complete the following information):**
 - Type:  Kubernetes
 - Version: 1.30
 - Provider: minikube

**Comment by bishalbera** [comment]
@kosmoz here is the output of the `make cert` 
<img width="1710" alt="Screenshot 2024-06-18 at 2 33 05 PM" src="https://github.com/glasskube/glasskube/assets/123734227/39d50542-7e19-44c4-a4fb-638d458154f6">

**Comment by kosmoz** [comment]
OK so that part works as expected at least

**Comment by jakepage91** [comment]
Hey there, I was able to recreate this issue. It turns out that [MacOS does some user level tmp folder division](https://apple.stackexchange.com/questions/353832/why-is-mac-osx-temp-directory-in-weird-path) which conflicts with how the tmp folder is then retrieved. The certs are actually located in the correct folder. 
We found a generic solution that might be a an easy fix, will leave it [here](https://unix.stackexchange.com/questions/174817/finding-the-correct-tmp-dir-on-multiple-platforms) for either @kosmoz or @christophenne
